src/views/vehicle/spawn/view_persist_car.cpp

```diff
@@ -135,7 +135,7 @@ namespace big
 					}
 
 					ImGui::SameLine();
-					if (ImGui::SmallButton("X"))
+					if (ImGui::SmallButton(std::string("X##" + pair).c_str()))
 					{
 						file_name_to_delete = pair;
 					}
```

Comment:
Probably best to use ImGui::PushID and ImGui::PopID functions.

```cpp
ImGui::PushID(pair_lower.c_str());
ImGui::SameLine();
if (ImGui::SmallButton("X"))
{
	file_name_to_delete = pair;
}
ImGui::PopID();
```

And example code in the codebase:
https://github.com/YimMenu/YimMenu/blob/990946119f81420bc1d38d0f1a2d0b580dd6ad82/src/views/settings/view_reaction_settings.cpp#L9-L12

Reply:
There is no int to use that's readily available, I'd have to make one, for something that is only used situationally.

Reply:
PushID supports char array and int.

Reply:
So it does. Fixed.